### PR TITLE
provide the video link of the object detection app

### PR DESCRIPTION
**Pull request by changlinzhang**

### CODE REVIEW

**Comment by changlinzhang** [comment]
solve [issue 240](https://github.com/intel-analytics/zoo/issues/240)

**Comment by changlinzhang** [comment]
@jenniew 
would you please help me review?

File: apps/object-detection/README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ This is a simple example of object detection using Analytics Zoo Object Detectio
 * Download Analytics Zoo and build it.
 * Run `export SPARK_HOME=the root directory of Spark`.
 * Run `export ANALYTICS_ZOO_HOME=the dist directory under the Analytics Zoo project`.
-* Prepare the video to detect. ([YouTube-8M](https://research.google.com/youtube8m/)).
+* Prepare the video to detect. ([YouTube-8M](https://research.google.com/youtube8m/) and [The video](https://www.youtube.com/watch?v=akcYAuaP4jw)).
```
Comment (jason-dai) [inline apps/object-detection/README.md]: Prepare [the video](...) in [YouTube-8M](...) to detect.

Comment (jenniew) [reply apps/object-detection/README.md]: Add instruction of how to download the video with the video link.

File: apps/object-detection/README.md [hunk 2]
```diff
@@ -10,7 +10,8 @@ This is a simple example of object detection using Analytics Zoo Object Detectio
 * Download Analytics Zoo and build it.
 * Run `export SPARK_HOME=the root directory of Spark`.
 * Run `export ANALYTICS_ZOO_HOME=the dist directory under the Analytics Zoo project`.
-* Prepare the video to detect. ([YouTube-8M](https://research.google.com/youtube8m/)).
+* Prepare the video in YouTube-8M to detect. ([YouTube-8M](https://research.google.com/youtube8m/) and [The video](https://www.youtube.com/watch?v=akcYAuaP4jw)).
+You can use a third-party tool, such as [savefrom](https://en.savefrom.net/), to download this video via its link.
```
Comment (jason-dai) [inline apps/object-detection/README.md]: I don't think we need to provide instructions for the third-party tool

**Comment by jason-dai** [comment]
Please update per comments and merge @helenlly

**Comment by jenniew** [comment]
LGTM

**Comment by jenniew** [comment]
We don't need to provide instruction to download as it may cause license issue.

**Comment by changlinzhang** [comment]
solved https://github.com/intel-analytics/zoo/issues/295 on master

**Comment by shane-huang** [comment]
Please kindly test it under pip install as well.

**Pull request merged**